apps/editing-toolkit/editing-toolkit-plugin/wpcom-global-styles/index.php

```diff
@@ -13,15 +13,13 @@
  */
 function wpcom_should_limit_global_styles( $blog_id = 0 ) {
 	if ( ! $blog_id ) {
-		$blog_id = get_current_blog_id();
-	}
-
-	// Do not limit Global Styles on Atomic sites for now, because blog stickers are not exposed
-	// to these sites and the project is still in a development stage that requires sites to have
-	// a certain blog sticker before restricting the feature. This is a temporary check that will
-	// be removed as part of the public launch.
-	if ( ! defined( 'IS_WPCOM' ) || ! IS_WPCOM ) {
-		return false;
+		if ( defined( 'IS_WPCOM' ) && IS_WPCOM ) {
+			$blog_id = get_current_blog_id();
+		} elseif ( method_exists( 'Jetpack_Options', 'get_option' ) ) {
```

Comment:
Nitpick: Should we make this condition more explicit? Instead of checking if the method exists, maybe we should check if it's an AT site instead. A Jetpack_Options class could exist in other environments.

Reply:
I don't think it's needed. ETK is only intended to work on Simple or Atomic sites. The reason why I added the check is to account for cases where Jetpack might be deactivated due to a broken connection or something like that.

Reply:
Looking at the existing usages of `Jetpack_Options` over ETK and wpcomsh, seems that most of them assume it's guaranteed to exist, so we shouldn't really check if the method exists. However, [the class was missing on an unit test which cause it to fail](https://github.com/Automattic/wp-calypso/actions/runs/3648257825/jobs/6161460937), so I think I'll follow your suggestion of checking whether the site is Atomic.